project/BuildHelper.scala

```diff
@@ -25,7 +25,7 @@ object BuildHelper {
   val Scala213: String = versions("2.13")
   val Scala3: String   = versions("3")
 
-  val SilencerVersion = "1.7.12"
+  val SilencerVersion = "1.7.14"
```

Comment:
You don't need Silencer anymore now that you don't support Scala 2.11 anymore. Scala's `@nowarn` annotation is sufficient